docs/dunst.pod

```diff
@@ -789,6 +828,10 @@ Or pushback which is equivalent to delay with the difference that already
 existing notifications are paused and hidden until the focus to the fullscreen
 window is lost.
 
+See B<layer> to change fullscreen behavior in Wayland
+
+Default: show
```

Comment:
Default layer was changed to `overlay`

Reply:
This section is about the fullscreen setting, which is x11 only. There is a reference to make users aware of the similar setting for wayland.

Reply:
Whoops, I misread that.